packages/terra-functional-testing/lib/terra-dev-site/doc/asset-service/AssetService.doc.mdx

```diff
@@ -0,0 +1,199 @@
+# Asset Service
+
+The asset service initializes a server to serve compiled assets to the testing environment.
+
+## Installation
+
+The asset service should be installed as a development dependency.
+
+```bash
+npm install --save-dev @cerner/terra-functional-testing
+```
+
+## Configuration
+
+To use the service it must be added to the services array in the `wdio.conf.js` file.
+
+```js
+// wdio.conf.js
+const AssetService = require('@cerner/terra-functional-testing/lib/services/wdio-asset-service');
+
+export.config = {
+    // ...
+    services: [
+        [AssetService, {
+            // Asset service options here.
+            // ...
+        }]
+    ],
+    // ...
+};
+```
+
+## Options
+
+### config
+
+A path to a webpack configuration file.
+
+Note: The `site` configuration option will take precedence over `config` if both are provided.
+
+Type: `string`
+
+Required: `false`
+
+Default: The asset service will check the current working directory for a `webpack.config.js` file.
+
+Example:
+
+```js
+// wdio.conf.js
+const AssetService = require('@cerner/terra-functional-testing/lib/services/wdio-asset-service');
+
+export.config = {
+    // ...
+    services: [
+        [AssetService, {
+          config: './path/webpack.config.js'
+        }]
+    ],
+    // ...
+};
+```
+
+### index
+
+The asset index.
+
+Type: `string`
+
+Required: `false`
+
+Default: `index.html`
+
+Example:
+
+```js
+// wdio.conf.js
+const AssetService = require('@cerner/terra-functional-testing/lib/services/wdio-asset-service');
+
+export.config = {
+    // ...
+    services: [
+        [AssetService, {
+          index: 'index.html',
+        }]
+    ],
+    // ...
+};
+```
+
+### locale
+
+An optional locale that will be passed into the webpack configuration as an environment variable.
+
+Type: `string`
+
+Required: `false`
+
+Default: `undefined`
```

Comment:
shouldn't this be english?

Reply:
There's a note in the existing service to avoid defaulting this value:

> // Explicitly not providing a fallback locale. Providing a fallback will lock the locale for all test runs when using the tt-wdio-runner.

https://github.com/cerner/terra-toolkit-boneyard/blob/master/src/wdio/services/ServeStaticService.js#L105

Reply:
Could we carry that note forward in the service?

Reply:
I'll add more documentation into this mdx doc, there's really no-where in the service this would fit into, but it could potentially be moved into the servers.js files

Reply:
yeah...forgot about that. I think it's a bit odd. Maybe the tt-wdio service needs to be updated or changes to prevent locking that value in if that is the case. I seems a little shady for this service to rely on undefined to be english since the screenshot naming falls back to `en`: https://github.com/cerner/terra-toolkit-boneyard/blob/main/config/wdio/visualRegressionConf.js#L56. Though maybe you are changing this in the next PR?

Reply:
I've left the default locale as undefined for now, I'll look more closely at this when I begin carrying forward the visual regression functionality and see if we can default it to 'en' without introducing any issues